central/policy/service/service_impl.go

```diff
@@ -347,6 +347,8 @@ func (s *serviceImpl) SubmitDryRunPolicyJob(ctx context.Context, request *storag
 	}
 
 	t := func(c concurrency.ErrorWaitable) (interface{}, error) {
+		// Copy over SAC scopes into background context
+		ctx := sac.CopyGlobalAccessScopeChecker(context.Background(), ctx)
```

Comment:
Note, there also already exists a generic function for stuff like this, not SAC-specific:
```suggestion
		ctx := contextutil.WithValuesFrom(context.Background(), ctx)
```
from `pkg/contextutil`

Reply:
😍 thx will use it